src/commonStyles.tsx

```diff
@@ -24,6 +24,58 @@ const styles = StyleSheet.create({
     fontSize: FONT_SIZE,
     fontWeight: '700',
   },
+
+  safeArea: {
+    backgroundColor: '#fff',
+  },
+
+  topBarButton: {
+    height: 25,
+    width: 25,
+    marginLeft: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+
+  container: {
+    flex: 1,
+  },
+
+  generalWrapper: {
+    paddingLeft: 20,
+    paddingRight: 20,
+  },
+
+  pane: {
+    padding: 5,
+    borderRadius: 5,
+    backgroundColor: '#fff',
+    elevation: 5,
+  },
+
+  titleLarge: {
+    fontFamily: FONT_FAMILY,
+    color: BLUE_TEXT,
+    fontSize: 40,
+    fontWeight: '700',
+    lineHeight: 50,
+    paddingLeft: 45,
+    paddingRight: 45,
+    textAlign: 'center',
+  },
+
+  formTitle: {
+    marginTop: 70,
+    marginBottom: 70,
```

Comment:
it can be `marginVertical` https://reactnative.dev/docs/layout-props#marginvertical